Resources/Private/Fusion/Override.fusion

```diff
@@ -0,0 +1,21 @@
+prototype(Neos.Neos:Page) {
+    googleAnalytics = Neos.GoogleAnalytics:TrackingCode.GA {
```

Comment:
I'm thinking about wrapping these 4 objects into one parent that could potentially being cached for the whole site depending on the setup. What do you think?

Reply:
See ad2cac27b60b756420f6cc9b715728a529f6a3fa. Like this?